### ISSUE TITLE
Downloadable assets main.scss file contains frontmatter

### ISSUE THREAD

**Issue by maya** [18F/web-design-standards#955]
In the downloadable assets in `assets/css/main.scss`, file contains frontmatter from jekyll. This should be removed so users do not get this.

This is what's currently in the file:

```
---
# Front matter comment to ensure Jekyll properly reads the file.
---

// Place list of your partial imports below, located in assets/_scss.

@import 'all';
```

**Comment by BrianDiggs** [comment]
There is similar front matter in `assets/css/stylesheet.css`

```
--- 
--- 
```

(My apologies if creating a new issue would have been more appropriate; it is very closely related and this one was still "in progress" .)

**Comment by maya** [comment]
@BrianDiggs thanks for your note! There is no  `assets/css/stylesheet.css` included in the downloadable assets zip file. Feel free to open another issue if you still find a problem.

**Comment by maya** [comment]
Closed via #956.
